app/assets/stylesheets/govuk-component/_taxonomy-sidebar.scss

```diff
@@ -1,29 +1,52 @@
 .govuk-taxonomy-sidebar {
-  border-top: 10px solid $mainstream-brand;
-  padding-top: 5px;
   @include core-16;
+  border-top: 2px solid $govuk-blue;
+  padding-top: $gutter-half;
+}
+
+.govuk-taxonomy-sidebar__related-heading,
```

Comment:
Non blocking:

These classes look to be the same, if we decouple how we present parts of this component from the semantics of this component, could these be change to a single '.govuk-taxonomy-sidebar__heading`?